src/accounts/context/userAccount.tsx

```diff
@@ -103,15 +111,25 @@ export const UserAccountProvider: FC<Props> = React.memo(({children}) => {
     }
   }, [dispatch, defaultAccountId])
 
-  async function handleSetDefaultAccount(newDefaultAcctId) {
+  async function handleSetDefaultAccount(
+    newDefaultAcctId: number,
+    setDefaultAccountOptions?: SetDefaultAccountOptions
+  ) {
     const accountName = getAccountNameById(newDefaultAcctId)
 
     try {
       await updateDefaultQuartzAccount(newDefaultAcctId)
       setDefaultAccountId(newDefaultAcctId)
-      dispatch(notify(accountDefaultSettingSuccess(accountName)))
+
+      if (setDefaultAccountOptions?.disablePopUps !== true) {
+        dispatch(notify(accountDefaultSettingSuccess(accountName)))
+      }
     } catch (error) {
-      dispatch(notify(accountDefaultSettingError(accountName)))
+      if (setDefaultAccountOptions?.disablePopUps !== true) {
+        dispatch(notify(accountDefaultSettingError(accountName)))
+      } else {
```

Comment:
When disabling popups from `handleSetDefaultAccount`, the desired behavior in the case of an error is for an error to be thrown to the User Profile page, so that the user profile page can catch it, and dispatch a single error notification.